Packs/SingleConnect/README.md

```diff
@@ -1,9 +1,9 @@
 Single Connect is an SAPM product that enables enterprises to remove static passwords stored in applications by instead keeping passwords in a secure password vault. Single Connect provides a token-based authentication for 3rd party applications when accessing the password vault. This authentication process verifies the application identity and gives secure access to the password associated with that identity.
 
 ## What does this pack do?
-This integration provides the following commands for SAPM:
+This integration provides the following commands for Shared Account Password Management (SAPM):
```

Comment:
```suggestion
This pack includes commands for Shared Account Password Management (SAPM):
```

Reply:
Hi @ostolero , 
Thank you, I've applied your suggestion